### ISSUE TITLE
Can I get the information about the GPU utilization of each user?

### ISSUE THREAD

**Issue by LinghaoChan** [wookayin/gpustat#113]
Can I get the information about the GPU utilization of each user?

**Comment by Stonesjtu** [comment]
I don't think nvidia-ml library can report GPU utilization per user. Do you find a solution for that?

**Comment by LinghaoChan** [comment]
No. My leader asked me to write a shell script to monitor everyone's GPU utilization. However, I have no idea and I am looking for a solution for it.

**Comment by wookayin** [comment]
I think this can be implemented using `nvmlDeviceGetProcessUtilization` API, which reports encoder/decoder/compute utilization per process.
